### PR TITLE
Add left handed EN messagease

### PR DESCRIPTION
**Pull request by BlueDrink9**
Additionally refactors out the ABC button as a common key; let me know if you'd rather I undo that or extend it to shift, BS and return

Partially addresses #661

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt [hunk 1]
```diff
@@ -32,6 +32,18 @@ import com.dessalines.thumbkey.utils.SlideType
 import com.dessalines.thumbkey.utils.SwipeDirection
 import com.dessalines.thumbkey.utils.SwipeNWay
 
+val ABC_KEY_ITEM =
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt:35]: Good idea. Could you grep for all these, probably with the term `Icons.Outlined.Abc` . I'm sure theres a lot more.

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt [hunk 2]
```diff
@@ -117,85 +119,87 @@ enum class KeyboardLayout(val index: Int, val keyboardDefinition: KeyboardDefini
...
-    FIMessageEase(88, KB_FI_MESSAGEEASE),
-    ENDAThumbKeySymbols(89, KB_EN_DA_THUMBKEY_SYMBOLS),
-    FIEEMessageEase(90, KB_FI_EE_MESSAGEEASE_SYMBOLS),
-    ENPLThumbkey(91, KB_EN_PL_THUMBKEY),
-    BGPhoneticMessagEaseSymbols(92, KB_BG_PHONETIC_MESSAGEASE_SYMBOLS),
-    UKMessageEaseSymbols(93, KB_UK_MESSAGEEASE_SYMBOLS),
-    ENFRMessagEaseSymbols(94, KB_EN_FR_MESSAGEASE_SYMBOLS),
-    HEMessageEaseSymbols(95, KB_HE_MESSAGEEASE_SYMBOLS),
-    ENNOMessagEaseSymbols(96, KB_EN_NO_MESSAGEASE_SYMBOLS),
+    ENMessageEaseLeft(16, KB_EN_MESSAGEEASE_LEFT),
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:122]: Revert this change. All new layouts need to be added to the end with a higher index.

**Review by dessalines — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt (hunk 2 above)
Comment (BlueDrink9) [reply app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt:122]: Ah I see, otherwise updates mess with it. Sure

**Comment by dessalines** [comment]
See https://github.com/dessalines/thumb-key/issues/661#issuecomment-1936701497

So it'd probably be best to leave the function, but add a setting.

**Comment by dessalines** [comment]
Also don't stack PRs please. If one builds on another, mark that one as draft until your first PR is merged.

**Comment by BlueDrink9** [comment]
The UI change to add the setting and translations etc is a bit beyond me at the moment, will need to wait until I have more time. I'll open a draft PR for it, and let this one get merged as separate layouts for now

**Comment by BlueDrink9** [comment]
ci failed due to network error

**Review by dessalines — APPROVED** [review]
